zeppelin-interpreter/src/main/java/com/nflabs/zeppelin/scheduler/Job.java

```diff
@@ -237,4 +238,8 @@ public Date getDateFinished() {
   private Logger logger() {
     return LoggerFactory.getLogger(Job.class);
   }
+
+  protected void setResult(InterpreterResult result) {
```

Comment:
Could you make setResult(Object result) instead of setResult(InterpreterResult result)?
Because of Job is designed to allow any java object as a result.